copier/cli.py

```diff
@@ -168,6 +171,21 @@ def data_switch(self, values: StrSeq) -> None:
             key, value = arg.split("=", 1)
             self.data[key] = value
 
+    @cli.switch(
+        ["--data-file"],
+        cli.ExistingFile,
+        help="Load data from a YAML file",
+        excludes=["--data"],
+    )
+    def data_file_switch(self, path: LocalPath) -> None:
```

Comment:
Wouldn't the type hint `cli.ExistingFile` be more adequate?

```suggestion
    def data_file_switch(self, path: cli.ExistingFile) -> None:
```

I missed this in the previous review. In case you agree, don't forget to remove the corresponding import.

Reply:
addressed in https://github.com/copier-org/copier/pull/1325/commits/f841be471d7dd4d229480c446b42820a82663517